style.css

```diff
@@ -0,0 +1,3 @@
+.fav-color {
+  color: blue;
+}
```

Comment:
Please associate your styles with a particular `element/tag` in your `.html` files or you can remove them if you're not using them in any file. Because leaving unused code in your codebase is not advisable.